jpo-ode-svcs/src/test/java/us/dot/its/jpo/ode/importer/parser/RxMsgFileParserTest.java

```diff
@@ -43,7 +46,7 @@
    @Test
    public void testStepsAlreadyDone() {
 
-      ParserStatus expectedStatus = ParserStatus.COMPLETE;
+      ParserStatus expectedStatus = ParserStatus.INIT;
```

Comment:
Repeat comment about whether the parser should return COMPLETE or INIT when it is finished.